src/App.tsx

```diff
@@ -1,11 +1,23 @@
-import React from 'react';
-
-import './App.css';
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+//Components
+import MainPage from "./pages/MainPage";
+import MainMenu from "./pages/GameOverMenu";
+import CreateLevel from "pages/CreateLevel";
```

Comment:
u have pages/... and ./pages/... . Use one style on project for component on top level